catalogs/sources/gtfs/schedule/at-unknown-bean-shuttle-gtfs-859.json

```diff
@@ -18,6 +18,8 @@
     },
     "urls": {
         "direct_download": "http://gtfs.beanshuttle.com/google_transit.zip",
+        "authentication_type": 0,
         "latest": "https://storage.googleapis.com/storage/v1/b/mdb-latest/o/at-unknown-bean-shuttle-gtfs-859.zip?alt=media"
-    }
-}
\ No newline at end of file
+    },
+    "redirect": 2036
```

Comment:
Failing integration tests because the redirect is not an array: 

"redirect": [
        {
            "id": 2036
        }
    ]